starter-code/index.html

```diff
@@ -2,30 +2,52 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-  Instagram
+    <div class="backgroundG">
+      </div>
+<div class="container">
+    
+    
+      <img class="phone" src="./images/phones.png">
+      <div class="textbox">
+      <header>
+          <h1><img class="logo" src="./images/brand.png"></h1>
```

Comment:
the h1 here is not necessary! You can simply remove it to make the login button and the logo aligned perfectly!